### PR TITLE
clang format and clang warnings

### PR DESCRIPTION
**Pull request by shmsong**
Just a format and warning cleanup PR.

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/lower_loops.h [hunk 1]
```diff
@@ -58,9 +58,6 @@ class TORCH_CUDA_CU_API LoopNestGenerator {
   // stack of the active for_loops
   std::vector<kir::ForLoop*> for_loops_;
 
-  // How many loops can the next iteration close
```
Comment (shmsong) [inline torch/csrc/jit/codegen/cuda/lower_loops.h:61]: @csarofeen This one seems to be from #1056, and it is currently unused. Just wondering if there was anything missing or we could just remove it.

Comment (csarofeen) [reply torch/csrc/jit/codegen/cuda/lower_loops.h:61]: I think this can be removed.

**Comment by csarofeen** [comment]
Sorry, should have merged this sooner, this likely now conflicts with https://github.com/csarofeen/pytorch/pull/1232

**Comment by shmsong** [comment]
> Sorry, should have merged this sooner, this likely now conflicts with #1232

No worries at all. This is an easy one to rebase : ).

**Review by csarofeen — APPROVED** [review]

**Pull request merged**